src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -98,6 +100,43 @@ export default function FormNav(props) {
     : `Step ${currentChapterDisplay} of ${chaptersLengthDisplay}: ${chapterName ||
         ''}`;
 
+  const handleFocus = async () => {
+    let root = document.querySelector('#react-root');
```

Comment:
would it be fine to just leave this as document? the fewer changes to looking up selectors the better to ensure no regressions, especially with custom HTML